src/ctap/ctap1.rs

```diff
@@ -571,13 +574,7 @@ mod test {
         let mut message =
             create_authenticate_message(&application, Ctap1Flags::CheckOnly, &key_handle);
 
-        message.push(0x00);
-        let response = Ctap1Command::process_command(&message, &mut ctap_state, START_CLOCK_VALUE);
-        assert_eq!(response, Err(Ctap1StatusCode::SW_WRONG_LENGTH));
-
-        // Two extra zeros are okay, they could encode the expected response length.
-        message.push(0x00);
-        message.push(0x00);
+        message.extend_from_slice(&[0x00, 0x00, 0x00, 0x00]);
```

Comment:
Before, we were checking if both types work: 1 or 3 zeros. Now, you are checking only once, and with 4 zeros. Was the unit test wrong before? And why not check both cases?

Reply:
@kaczmarczyck before any extra bytes were bad. Now, there might be 1-3 additional bytes at the end, so the old test will now simply be parsed as Le=0. Appending four bytes to really overflow length.

Reply:
Okay, can you add tests for all different possible lengths then, to at least match the test strength from before?

Reply:
@kaczmarczyck done